front/pages/w/[wId]/builder/data-sources/[name]/upsert.tsx

```diff
@@ -283,213 +284,123 @@ export default function DataSourceUpsert({
       }
       hideSidebar={true}
     >
-      <div className="flex flex-col pt-8">
-        <div className="space-y-2">
-          <div className="flex flex-1">
-            <div className="w-full">
-              <div className="grid gap-x-4 gap-y-4 sm:grid-cols-5">
-                <div className="sm:col-span-5">
-                  <label
-                    htmlFor="documentId"
-                    className="block text-sm font-medium text-gray-700"
-                  >
-                    Document ID
-                  </label>
-                  <div className="mt-1 flex rounded-md shadow-sm">
-                    <input
-                      type="text"
-                      name="document"
-                      id="document"
-                      readOnly={readOnly}
-                      className={classNames(
-                        "block w-full min-w-0 flex-1 rounded-md border-gray-300 text-sm",
-                        "border-gray-300",
-                        readOnly
-                          ? "focus:border-gray-300 focus:ring-0"
-                          : "focus:border-action-500 focus:ring-action-500"
-                      )}
-                      value={documentId}
-                      onChange={(e) => setDocumentId(e.target.value)}
-                    />
-                  </div>
-                  <p className="my-2 text-sm text-gray-500">
-                    The ID of the document (it can be anything such as a file
-                    name or title). Upserting with the ID of a document that
-                    already exists will replace it.
-                  </p>
-                </div>
-              </div>
+      <div className="pt-6">
+        <Page.Vertical align="stretch">
+          <div className="pt-4">
+            <Page.SectionHeader title="Document title" />
+            <div className="pt-4">
+              <Input
+                placeholder="Document title"
+                name="document"
+                disabled={readOnly || !!loadDocumentId}
+                value={documentId}
+                onChange={(v) => setDocumentId(v)}
```

Comment:
We don't need the target.value?

Reply:
sparkle returns the value directly it seems